src/ai/backend/manager/registry.py

```diff
@@ -3023,7 +3027,11 @@ async def get_commit_status(
         session_name_or_id: Union[str, SessionId],
         access_key: AccessKey,
     ) -> Mapping[str, str]:
-        kernel = await self.get_session(session_name_or_id, access_key)
+        kernel = await self.get_session(session_name_or_id, access_key, allow_stale=True)
+        if kernel["status"] != KernelStatus.RUNNING:
+            raise InvalidAPIParameters(
+                f'Unable to get commit status since kernel(id: {kernel["id"]}) is currently not RUNNING.'
+            )
```

Comment:
Please use double quotation mark! 
```suggestion
                f"Unable to get commit status since kernel(id: {kernel["id"]}) is currently not RUNNING."
            )
```